.github/workflows/release.yml

```diff
@@ -0,0 +1,53 @@
+name: Release
+
+# Runs manually
+on:
+  workflow_dispatch:
+      
+jobs:
+  release:
+    name: Release on Sonatype OSS
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up Apache Maven Central
+        uses: actions/setup-java@v3
+        with:
+          java-version: 11
+          distribution: 'temurin'
+          cache: 'maven'
+          gpg-private-key: ${{ secrets.GPG_PRIVATE_KEY }} # Value of the GPG private key to import
+          gpg-passphrase: GPG_PASSPHRASE # env variable for GPG private key passphrase
+
+      - name: Publish to Apache Maven Central
+        run: |
+          maven_config="$(cat .mvn/maven.config)"
+          snapshot_version="${maven_config%% *}"
+          suffix="-SNAPSHOT"
+          release_version=${snapshot_version%"$suffix"}
+          sed -i "s/${snapshot_version}/${release_version}/" .mvn/maven.config
+          mvn --settings .mvn/settings.xml deploy -Pdeploy,release -Dgpg.passphrase=${{ secrets.GPG_PASSPHRASE }}
+        env:
+          SONATYPE_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
+          SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
+          GPG_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
+
+      # Create github release with the specified tag, name and body
+      - name: create gitHub release
+        run: |
+          maven_config="$(cat .mvn/maven.config)"
+          version="${maven_config%% *}"
+          prefix="-Drevision="
+          release_version=${version#"$prefix"}
+          noDotVersion="${release_version//.}"
+          gh release create ${version} --notes "# Download
+          https://repo1.maven.org/maven2/com/devonfw/tools/ide/devonfw-ide-scripts/${version}/devonfw-ide-scripts-${version}.tar.gz
+          # Changes
+          https://github.com/devonfw/ide/blob/master/CHANGELOG.asciidoc#${noDotVersion}"
+          git tag Release/"${version}"
```

Comment:
See also this documentation of `release` commandlet:
https://github.com/devonfw/ide/blob/master/documentation/release.asciidoc#release

Reply:
And for SCM best practices, the tag should be annotated:
https://github.com/devonfw/ide/blob/d250ffa520daa74bf0068737c5c0f6191c93fe95/scripts/src/main/resources/scripts/command/release#L106

Reply:
Also it is with lowercase `release/` insteadof `Release/`.

Reply:
> Also what confuses me is that you are doing this `maven.config` tweaks twice. Why is this required? Are the effects from the previous steps lost here?
Yes, because those are two different steps and therefore each run creates a new script. The variables of the first script are not available for the second. I could do everything by running large script but I think this approach with different named steps is better. One solution would be using source but I just created a file to save the variable and read it again in the next step. I hope that's okay.

Reply:
I compute the next SNAPSHOT version automatically avoiding an error-prone parameter.
Did also some fixed to tag and commit comments and added proper title to github release.
Still needs testing but should be resolved now.